pkg/node/provisioners/utilwrappers/partition_operations.go

```diff
@@ -155,6 +158,25 @@ func (d *PartitionOperationsImpl) ReleasePartition(p Partition) error {
 	return nil
 }
 
+// isMounted judge if device like /dev/sda is mounted or not
+func isMounted(device string) bool {
+	// remove the prefix /dev as the mountinfo only contains suffix.
+	device = strings.Trim(device, "/dev")
+	procMounts, err := util.ConsistentRead("/proc/self/mountinfo", 5, time.Millisecond)
```

Comment:
Hi @mishoyama,  do you have suggestions for this kinds of test coverage ?  since it is reading a file in host,  I could not mock the contents of this file,  so test coverage can't be 100% covered I think.

Reply:
add test for this function,  reading host /proc/self/mountinfo,  as any machine should contain this file. 
mock a failure and success case here.   
for success case,  /proc entry should included in every linux machine, so it will return true. 
and for failure case,  we use /dev/nvme50n1,  there should not have such device generally.